docs-kits/kits/Connector Kit/Adoption View/working-with-policies.md

```diff
@@ -0,0 +1,288 @@
+---
+sidebar_position: 2
+description: 'General Information on EDC Policies'
+title: Working with Policies
+id: connector_kit_adoption_view_policies
+---
+
+## Working with Policies
+
+Policies are a formalization of the terms under which a Provider extends an offer to the Dataspace. The format is given
+by the Dataspace Protocol Specification: The [Open Digital Rights Language (ODRL)](https://www.w3.org/TR/odrl-model/).
+
+Like all payloads that get exchanged between connectors, ODRL is an RDF-based description language that is on exchange
+serialized as JSON-LD. JSON-LD is namespace-aware JSON with a couple of twists that one should be aware of when working
+with it (like "Structures may be semantically equivalent even though, schematically, they are clearly not"). ODRL
+enables a Provider to be very specific when it comes to the usage of data: What may a consumer do with it? What
+obligations does a consumer incur when accepting the offer?
+
+The first time the Consumer encounters a Policy is when evaluating a Provider's [Catalog](https://docs.internationaldataspaces.org/ids-knowledgebase/v/dataspace-protocol/catalog/catalog.protocol), as displayed below.
+This specific Catalog contains one single `dcat:Dataset`, called "json-1-paper". It is the only entry in the top-level
+`dcat:dataset` property. To access this Dataset, the Consumer can choose between four Offers (see the `odrl:hasPolicy`
+property):
+
+- "Y29udHJhY3QtYmlsYXRlcmFsLXBhcGVyLWV4YW1wbGUtNA==:anNvbi0xLXBhcGVy:Y2JhMGFmM2QtZTIwYi00ZjExLThlZmQtM2FhMzcyZmI1Nzhh"
+- "Y29udHJhY3QtYmlsYXRlcmFsLXBhcGVyLWV4YW1wbGUtMg==:anNvbi0xLXBhcGVy:ZDA4ZDM5OTgtOGY5ZS00MzBmLThjZDEtZmYwOWQxMmQxYzk5"
+- "Y29udHJhY3QtYmlsYXRlcmFsLXBhcGVyLWV4YW1wbGUtMw==:anNvbi0xLXBhcGVy:NDQ1ZWRlZDMtNjNlMy00ZDg0LTkxMmUtYWFjZWM0MjdkMDQ4"
+- "Y29udHJhY3QtYmlsYXRlcmFsLXBhcGVyLWV4YW1wbGUtMQ==:anNvbi0xLXBhcGVy:ODFkMDI2MWYtNDNlNi00ZTIxLWJkMWYtZmFmZTI3MWQwYzhj"
+
+```json
+{
+  "@context": {
+    "@vocab": "https://w3id.org/edc/v0.0.1/ns/",
+    "edc": "https://w3id.org/edc/v0.0.1/ns/",
+    "tx": "https://w3id.org/tractusx/v0.0.1/ns/",
+    "dcat": "http://www.w3.org/ns/dcat#",
+    "dct": "https://purl.org/dc/terms/",
+    "odrl": "http://www.w3.org/ns/odrl/2/",
+    "dspace": "https://w3id.org/dspace/v0.8/"
+  },
+  "@id": "693e9b66-04f2-4bfb-b3cd-daf5857b47c9",
+  "@type": "dcat:Catalog",
+  "dcat:dataset": [
+    {
+      "@id": "json-1-paper",
+      "@type": "dcat:Dataset",
+      "odrl:hasPolicy": [
+        {
+          "@id": "Y29udHJhY3QtYmlsYXRlcmFsLXBhcGVyLWV4YW1wbGUtNA==:anNvbi0xLXBhcGVy:Y2JhMGFmM2QtZTIwYi00ZjExLThlZmQtM2FhMzcyZmI1Nzhh",
+          "@type": "odrl:Set",
+          "odrl:permission": {
+            "odrl:action": {
+              "odrl:type": "http://www.w3.org/ns/odrl/2/use"
+            },
+            "odrl:constraint": {
+              "odrl:and": [
+                {
+                  "odrl:leftOperand": "https://w3id.org/catenax/policy/v1.0.0/FrameworkAgreement",
```

Comment:
This seems to be not the latest:
https://w3id.org/catenax/policy/v1.0.0/FrameworkAgreement
should be:
https://w3id.org/catenax/policy/FrameworkAgreement
without the version...

Reply:
Ok, so just the rightOperand is versioned?
I assume that this change is adopted by the reference implementation since it probably string-matches the leftOperand to trigger a validation of the rightOperand against a VC.

Reply:
Even in CX-0018 it is without the v1.0.0:
https://github.com/catenax-eV/product-standardization-prod/blob/CX-0018-CX-Rel2405-WIP/standards/CX-0018-DataspaceConnectivity/3.0.0/CX-0018-DataspaceConnectivity-3.0.0.md#24-conventions-for-policy-constraints